compiler/runtime/OMRSharedCacheRelocationRuntime.cpp

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2019 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#include "runtime/OMRSharedCacheRelocationRuntime.hpp"
+
+TR::RelocationRuntime * OMR::SharedCacheRelocationRuntime::self()
+   {
+   return reinterpret_cast<TR::RelocationRuntime*>(this);
+   }
+
+void *
+OMR::SharedCacheRelocationRuntime::symbolAddress(char *symbolName)
+   {
+   std::string symbol(const_cast<const char *>(symbolName));
+   return _symbolLocation[symbol];
+   }
+
+void
+OMR::SharedCacheRelocationRuntime::registerLoadedSymbol(const char *&symbolName, void *&symbolAddress)
+   {
+   std::string method(symbolName);
+   _symbolLocation[symbolName] = symbolAddress;
+   }
```

Comment:
I think you need a new line at the end of the file.

Reply:
Thank you, I will fix that the next commit!